gulpfile.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const gulp = require('gulp');
+const responsive = require('gulp-responsive');
```

Comment:
I'm open to this if it's the only way we can make it work... but I'm hesitant to add `gulp` unless we have to. I'm wondering if we could use something like https://github.com/MarcusCemes/responsive-image-builder to do the same thing? I'd have to dig into it a bit more.

Reply:
I vote for merging this and seeking alternatives in another card for later; but could defer to @jgerigmeyer 's thoughts on that.